### PR TITLE
README: What to Tweet

### PR DESCRIPTION
**Pull request by gr2m**
-----
[View rendered README.md](https://github.com/probot/twitter/blob/gr2m-patch-2/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,21 @@
 
 Follow [these instructions](./tweets) for a detailed guide, but the short version is: create a `.tweet` file in `/tweets` and open a pull request. We'll review it, and once it's merged, the [@ProbotTheRobot](https://twitter.com/ProbotTheRobot) Twitter account will tweet it out!
 
+## What to Tweet
+
+We encourage anyone to craft and form a tweet, by opening a PR in the tweets folder. Possible topics include:
+
+- Interesting usage / us cases of Probot
```
Comment (JasonEtco) [inline README.md]:
```suggestion
- Interesting usage/use-cases of Probot
```

File: README.md [hunk 2]
```diff
@@ -6,6 +6,21 @@
...
+- Tutorials / projects GitHub workflow automation
```
Comment (JasonEtco) [inline README.md]:
```suggestion
- Tutorials/projects about GitHub workflow automation
```

File: README.md [hunk 3]
```diff
@@ -6,6 +6,21 @@
...
+- News about Probot and related projects
+- Other items relevant to the Probot community
+
+If you build an app with Probot, we are happy to share the word! The app should either be open soure or should mention that it’s built with Probot in the marketplace / app description as well as the website if one exists.
```
Comment (JasonEtco) [inline README.md]:
```suggestion
If you build an app with Probot, we are happy to share the word! The app should either be open source or should mention that it’s built with Probot, in the marketplace/app description as well as the website if one exists.
```

File: README.md [hunk 4]
```diff
@@ -6,6 +6,21 @@
...
+
+## Accepting a tweet
+
+One accept by a maintainers is enough for a merge.
```
Comment (JasonEtco) [inline README.md]:
```suggestion
One acceptance by a maintainer is enough for a merge.
```

**Comment by gr2m** [comment]
ping @probot/maintainers. Will merge later today unless there are objections, but we can do follow ups :)

**Comment by gr2m** [comment]
ping @JasonEtco  can’t merge until you approve :)

**Pull request merged**